flink-connectors/flink-connector-hbase-1.4/src/main/java/org/apache/flink/connector/hbase1/HBase1DynamicTableFactory.java

```diff
@@ -74,13 +72,13 @@ public DynamicTableSource createDynamicTableSource(Context context) {
         validatePrimaryKey(tableSchema);
         validateTableSourceOptions(tableOptions);
```

Comment:
We don't need this, this has been checked in `helper.validateExcept(PROPERTIES_PREFIX)`.

Reply:
validatePrimaryKey is necessary，The other can be removed.